src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
Ack. I'm not sure there is utility in having `gapic_package_name` and `proto_package_name` if they are always derivable from `package_name`. But just changing the semantics of `package_name` SGTM.